src/main/resources/doc/en/html/libs/gates/controlled.html

```diff
@@ -224,7 +224,7 @@ <h2>
           Facing
         </dt>
         <dd>
-          The direction of the component, its output relative to its input.
+          The direction of the component, its output relative to it input.

```

Comment:
That's not correct, so please roll back. And after a thought, please leave "its" as it was before. Sorry for the mess.

Reply:
The fault is on my side, I used directly GitHub web and it does not suit me.